internal/config/config_test.go

```diff
@@ -32,34 +32,46 @@ func TestInterpolateVars(t *testing.T) {
 	assert.EqualValues(t, cfg.MQTT.StandbyTopic, "cmd/test/standby/device/#")
 }
 
-func TestReadFromFile(t *testing.T) {
-	cfgNoPath := config.Config{
-		SiteName:     "test",
-		SerialNumber: "device",
-	}
-	_, err := cfgNoPath.NewFromFile()
-	assert.Error(t, err)
+func TestReadFromFile_WhenConfigFilePathExists_ReadsConfig(t *testing.T) {
+	testPath := "../../tests/integration/config.yaml"
+	require.FileExists(t, testPath)
 
-	cfgBadPath := config.Config{
+	t.Log(config.DumpYAML("../../tests/integration/config.yaml"))
```

Comment:
@xpatiate I missed that part - this could be `config.DumpYAML(testPath)`. Alternatively, this line may be deleted altogether - it's up to you.